seannkelleyy/Mario-Remake
# added sprites for underground section, piranha, and bullet.

﻿
# Description
added sprites for underground section, piranha, and bullet.

closes #85

## Type of change


- New feature (non-breaking change which adds functionality)

# How To Test

Check that branch still runs and check naming scemes of new sprites in the sprite variables section.

# Checklist:
